CONTRIBUTING.txt

```diff
@@ -46,6 +46,15 @@ Here's the long and short of it:
      - ``upstream``, which refers to the ``scikit-image`` repository
      - ``origin``, which refers to your personal fork
 
+.. note::
+
+    Although our code is hosted on `github
+    <https://github.com/scikit-image/>`_, our database is stored on `gitlab
```

Comment:
```suggestion
    <https://github.com/scikit-image/>`_, our datasets are stored on `gitlab
```

Reply:
database sounds like we have a big database like mysql etc.

Reply:
@sciunto you've marked this as resolved but I still see database?